includes/wc-core-functions.php

```diff
@@ -62,9 +62,7 @@
  * @param  string $value
  */
 function wc_maybe_define_constant( $name, $value ) {
-	if ( ! defined( $name ) ) {
-		define( $name, $value );
-	}
+	WC()->define( $name, $value );
```

Comment:
Not keen on this because it means adding public methods to maintain on main class. Would rather this logic was here too. I know thats duplicate though...